x-pack/plugins/elastic_assistant/server/types.ts

```diff
@@ -241,6 +242,7 @@ export interface AssistantToolParams {
   connectorId?: string;
   chain?: RetrievalQAChain;
   esClient: ElasticsearchClient;
+  esStore?: ElasticsearchStore;
```

Comment:
We can say _bye bye_ to this 🙂 👋  <img width="16" src="https://github.com/user-attachments/assets/666cd159-b896-41bb-8e5c-1aa2df4db689" />